### PR TITLE
musescore: fix directory issue

### PR DESCRIPTION
**Pull request by TURX**

### CODE REVIEW

File: media-sound/musescore/musescore-3.2.3.recipe [hunk 1]
```diff
@@ -65,17 +65,17 @@ BUILD_PREREQUIRES="
...
 {
-	make install PREFIX=$prefix UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
+	make install PREFIX=$appsDir UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
 		BUILD_JACK=OFF BUILD_PORTAUDIO=OFF BUILD_WEBENGINE=OFF \
 		USE_SYSTEM_FREETYPE=ON DOWNLOAD_SOUNDFONT=OFF BUILD_ALSA=OFF
-	rm -rf $prefix/share/icons $prefix/share/applications/mscore.desktop
-	mv $prefix/share $prefix/data
+	rm -rf $appsDir/share/icons $appsDir/share/applications/mscore.desktop
+	mv $appsDir/share $appsDir/data
 	addAppDeskbarSymlink $binDir/musescore Musescore
```
Comment (pulkomandy) [inline media-sound/musescore/musescore-3.2.3.recipe]: This link is now broken. Why move musescore to appsDir? It's a ported application so it uses the standard filesystem organization (with bin/ data/ etc) and I see no point in moving it around, except adding more maintenance effort?

Comment (TURX) [reply media-sound/musescore/musescore-3.2.3.recipe]: The modifications are for issue #3217.

File: media-sound/musescore/musescore-3.2.3.recipe [hunk 2]
```diff
@@ -65,17 +65,17 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make release PREFIX=$prefix UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
+	make release PREFIX=$appsDir UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
```
Comment (Begasus) [inline media-sound/musescore/musescore-3.2.3.recipe]: Haven't checked, but couldn't you add BINDIR=$appsDir here? No need to modify the other issues then?

File: media-sound/musescore/musescore-3.2.3.recipe (hunk 1 above)
Comment (pulkomandy) [reply media-sound/musescore/musescore-3.2.3.recipe]: If we really want to fix this, we could move just the executable from bin/ to apps/, but leave the other things where they are.

In any case, please fix the addAppDeskbarSymLink to point to the executable, and make sure it's possible to start the application from DeskBar.

**Comment by Begasus** [comment]
Still haven't checked it out yet, but I see on Github that they use cmake, maybe have a look there?

File: media-sound/musescore/musescore-3.0.0_git.recipe [hunk 3]
```diff
@@ -69,4 +69,6 @@ INSTALL()
 {
 	make install PREFIX=$prefix UPDATE_CACHE=FALSE
 	mv $prefix/share $prefix/data
+	mv $binDir/musescore $appsDir/musescore
+	addAppDeskbarSymlink $appsDir/musescore Musescore
```
Comment (Begasus) [inline media-sound/musescore/musescore-3.0.0_git.recipe]: Maybe:
```diff
-	mv $binDir/musescore $appsDir/musescore
-	addAppDeskbarSymlink $appsDir/musescore Musescore
+	mv $binDir/musescore $appsDir/Musescore
+	addAppDeskbarSymlink $appsDir/Musescore
```

File: media-sound/musescore/musescore-3.2.3.recipe [hunk 4]
```diff
@@ -77,5 +77,6 @@ INSTALL()
 		USE_SYSTEM_FREETYPE=ON DOWNLOAD_SOUNDFONT=OFF BUILD_ALSA=OFF
 	rm -rf $prefix/share/icons $prefix/share/applications/mscore.desktop
 	mv $prefix/share $prefix/data
-	addAppDeskbarSymlink $binDir/musescore Musescore
+	mv $binDir/musescore $appsDir/musescore
+	addAppDeskbarSymlink $appsDir/musescore Musescore
```
Comment (Begasus) [inline media-sound/musescore/musescore-3.2.3.recipe]: Maybe:
```diff
-	mv $binDir/musescore $appsDir/musescore
-	addAppDeskbarSymlink $appsDir/musescore Musescore
+	mkdir -p $appsDir
+	ln -s $binDir/mscore $appsDir/MuseScore
+	addAppDeskbarSymlink $appsDir/MuseScore
```
**EDIT** still checking the INSTALL()

File: media-sound/musescore/musescore-3.2.3.recipe (hunk 1 above)
Comment (Begasus) [reply media-sound/musescore/musescore-3.2.3.recipe]: Default install uses /usr/local, that is overridden by our PREFIX, but "share" is still being used for the data, either we use PREFIX=$appsDir/MuseScore or try to fix the "share" in the build/install part

**Comment by TURX** [comment]
Could you please comment with line numbers?

> Not needed here:
> 
> ```diff
> -	mkdir -p $appsDir
> ```

**Comment by Begasus** [comment]
> Could you please comment with line numbers?
> 
> > Not needed here:
> > ```diff
> > -	mkdir -p $appsDir
> > ```

Thought I mentioned them offending lines, will look again

File: media-sound/musescore/patches/musescore-3.0.0_git.patchset [hunk 5]
```diff
@@ -196,3 +196,32 @@ index bd71a62..eea59b1 100644
...
+ #include <QClipboard>
+-#include <QPlainTextEdit>
+ #include <QStyledItemDelegate>
+
+ #include <QDateTimeEdit>
+@@ -121,6 +120,7 @@
+ #include <QPrintDialog>
+ #include <QPrinter>
+ #endif
++#include <QPlainTextEdit>
```
Comment (korli) [inline media-sound/musescore/patches/musescore-3.0.0_git.patchset]: this shouldn't be needed.

File: media-sound/musescore/patches/musescore-3.2.3.patchset [hunk 6]
```diff
@@ -49,3 +49,32 @@ index 8033b35..5a8c314 100644
...
+ #include <QClipboard>
+-#include <QPlainTextEdit>
+ #include <QStyledItemDelegate>
+
+ #include <QDateTimeEdit>
+@@ -121,6 +120,7 @@
+ #include <QPrintDialog>
+ #include <QPrinter>
+ #endif
++#include <QPlainTextEdit>
```
Comment (korli) [inline media-sound/musescore/patches/musescore-3.2.3.patchset]: ditto

**Comment by korli** [comment]
merged, edited manually.

**Pull request closed without merging**